playbooks/upload_to_swift.yml

```diff
@@ -44,79 +49,81 @@
       tags:
         - skip_ansible_lint
 
-    - name: Create a public Cloud Files container
+    - name: Create a Cloud Files container
       os_object:
         container: "{{ container }}"
         container_access: "public"
         region_name: "{{ region }}"
         cloud: "{{ cloud_name }}"
+      register: _create_container
+      until: _create_container | success
+      retries: 5
+      delay: 10
 
     - name: Authenticate to the cloud and retrieve the service catalog
       os_auth:
         cloud: "{{ cloud_name }}"
         region_name: "{{ region }}"
       no_log: true
```

Comment:
Is this one left because otherwise it will still log sensitive stuff?

Reply:
Yes, if it fails it spits out a bunch of things which are sensitive, so I thought it better to leave this as no_log.